### ISSUE TITLE
Incorrect GraphQL URL for github enterprise

### ISSUE THREAD

**Issue by yulrizka** [microsoft/vscode-pull-request-github#1381]
- Extension version: 0.12.0
- VSCode Version:  1.40.0-insider
- OS: Osx
- Github Enterprise version: GitHub Enterprise Server 2.16.12

Steps to Reproduce:

1. Login with github enterprise
2. See the log message
```
[Info] https://github.[company].org/api/v3: GraphQL not supported (Network error: Response not successful: Received status code 404)
```
I think this is related to https://github.com/microsoft/vscode-pull-request-github/issues/947

This is caused by invalid graphQL URL for the enterprise version. The reason is that this function will fallback to `/api/v3` for URL other than github.com

https://github.com/microsoft/vscode-pull-request-github/blob/aae86fdfd2045e1250cef44782b08c047e9a6a85/src/authentication/configuration.ts#L18-L25

But when constructing the ApolloClient, it append the base url with `/graphql`. which the URL now becomes `/api/v3/graphql`. That's why it returns 404.

https://github.com/microsoft/vscode-pull-request-github/blob/aae86fdfd2045e1250cef44782b08c047e9a6a85/src/github/credentials.ts#L316-L320

Before fixing this, I couldn't create a pending PR review and comments. New comments was immediately posted as single comments (not as part of the pending review). I believe this is because it's using the REST API instead of the graphQL underneath.

How ever, once i use the correct graphQL URL. Loading Description page of PR no longer works. Got error with this 
```
[Debug 130796s 117ms] PullRequestOverviewPanel> initialized without PR
[Debug 130796s 320ms] GitHubRepository> Fetch available merge methods - done
[Debug 130796s 328ms] GithubRepository> Unable to fetch PR: Error: GraphQL error: Field 'isDraft' doesn't exist on type 
```
Not sure why that is. But at least now I can use Draft PR functionalities

**Comment by IllusionMH** [comment]
Looks like #1382 resolved this issue but didn't use `fixes #1381` to trigger auto-close.

**Comment by yulrizka** [comment]
It should, i might missed the '#' character in the PR title
